EIPS/eip-7503.md

```diff
@@ -0,0 +1,111 @@
+
+$y^2=x^3+7$
+
+Because of the discrete logarithm problem, it's practically impossible to calculate $s$ where 
+$s \times G$ is equal with our point.
+
+Let's say $R$ is an unspendable public-key (Let's call it the reference unspendable public-key). 
+People can publicly burn their tokens by sending them to $R$, after doing so, others can indeed 
+conclude that they have burnt their tokens, because they can all see the transactions and they 
+know that $R$ is unspendable.
+
+***We can derive infinitely many provably-unspendable public-keys from a reference unspendable public-key***
+
+Let's pick a random secret value $t$ and calculate a new point $D=R + t \times G$. We can prove 
+that $D$ is also unspendable, because $log_G(D)=log_G(R + t \times G)=log_G(R) + t$, and since 
+we can't calculate $log_G(R)$, the public-key $D$ is also unspendable.
+
+Obviously, $D$ is a completely new point that does not seem unspendable. We can convince others 
+that $D$ is unspendable by revealing $t$, because then people can verify that $D$ is the result 
+of adding some other point to $R$.
+
+***Using the help of Zero-Knowledge proofs, we can hide the value of $t$***
+
+We just need to prove that we know a secret value $t$ where $R + t \times G == D$. We can go even 
+further. We can prove that some EOA-to-EOA transaction has happened in the history of 
+blockchain, which has burnt some amount of ETH. By revealing this to the Ethereum blockchain and 
+providing something like a nullifier (E.g. hash of the secret value $t$ so that double minting of 
+same burnt tokens are not possible), we can add a new ***minting*** functionality for ETH so that
+people can migrate they secretly burnt tokens to a completely new address, without any trace on 
+the blockchain. This will make an anonymity pool of the size of the entire Ethereum blockchain 
+without much hassle.
+
+## Rationale
+
+Cryptocurrency mixers like TornadoCash can successfully obfuscate Ethereum transactions, but it's
+easy for the governments to ban usage of them. Anybody who has interactions with a mixer contract,
+whether the sender or receiver, can get marked. The EIP tries to at least minimize the privacy leakage 
+of the senders. The goal was to require zero smart-contract interactions in order to send money, so 
+we had to only use EOA-to-EOA transfers. In order to have a "teleportation" mechanism we had to divide
+the set of all Secp256k1 points $E(K)$ into two subsets/address-spaces:
+
+ - The spendable address-space: $\\{p \in E(K) | \exists s : g^s=p\\}$
+ - The unspendable address-space: $\\{p \in E(K) | \nexists s : g^s=p\\}$
+
+The spendable/unspendable addresses are not distinguishable, so we can exploit this fact and define
+an spendability rule for the money sent to addresses that can't be spent using regular elliptic-curve
+signatures. Using the help of Zero-Knowledge proofs, we can hide the transaction trace and design 
+a new privacy protocol, which is what this EIP is proposing.
```

Comment:
how do you stop double claims of the spend?

Reply:
By using a nullifer value (Just like TC), the nullifier can simply be the hash of the secret value.